Sources/MapboxMaps/Annotations/AnnotationOrchestrator.swift

```diff
@@ -77,6 +79,7 @@ public class AnnotationOrchestrator {
     ///   - layerPosition: Optionally set the `LayerPosition` of the layer managed.
     /// - Returns: An instance of `PolygonAnnotationManager`
     public func makePolygonAnnotationManager(id: String = String(UUID().uuidString.prefix(5)),
+                                             isPersistent: Bool = true,
```

Comment:
should this be `shouldPersist` instead of `isPersistent`. Wonder if it reads better..

Reply:
Fixed in code gen so it's consistent.